packages/web/src/pages/dashboard-page/components/ArtistContentSection.tsx

```diff
@@ -0,0 +1,179 @@
+      if (!isTracks && pill === Pills.TRACKS) {
+        setSelectedAlbumFilter(null)
+      } else if (isTracks && pill === Pills.ALBUMS) {
+        setSelectedTrackFilter(null)
+      }
+    },
+    [isTracks]
+  )
+
+  const {
+    filterButtonOptions: filterButtonTrackOptions,
+    data: tracks,
+    hasOnlyOneSection: hasOnlyOneTrackSection
+  } = useArtistDashboardTrackFilters({
+    selectedFilter: selectedTrackFilter,
+    filterText
+  })
+  const {
+    filterButtonOptions: filterButtonAlbumOptions,
+    data: albums,
+    hasOnlyOneSection: hasOnlyOneAlbumSection
+  } = useArtistDashboardAlbumFilters({
+    selectedFilter: selectedAlbumFilter,
+    filterText
+  })
+  const filterButtonOptions = isTracks
+    ? filterButtonTrackOptions
+    : filterButtonAlbumOptions
+  const shouldShowFilterButton =
+    (isTracks && !hasOnlyOneTrackSection) ||
+    (!isTracks && !hasOnlyOneAlbumSection)
+  const shouldShowPills = tracks.length && albums.length
+
+  if (!tracks.length && !albums.length) return null
+
+  return (
+    <Paper w='100%' direction='column' mt='xl'>
+      <Flex ph='2xl' pv='l' justifyContent='space-between'>
+        <Flex gap='2xl'>
+          {shouldShowPills ? (
+            <Flex gap='s'>
+              <SelectablePill
+                isSelected={selectedPill === Pills.TRACKS}
+                label={messages.tracks}
+                size='large'
+                onClick={() => onClickPill(Pills.TRACKS)}
+              />
+              <SelectablePill
+                isSelected={selectedPill === Pills.ALBUMS}
+                label={messages.albums}
+                size='large'
+                onClick={() => onClickPill(Pills.ALBUMS)}
+              />
+            </Flex>
+          ) : null}
+          {
+            // Only show filter button if there are multiple sections for the selected content type
+            shouldShowFilterButton ? (
+              <FilterButton
+                onSelect={handleSelectFilter}
+                selection={isTracks ? selectedTrackFilter : selectedAlbumFilter}
+                label={messages.allReleases}
+                options={filterButtonOptions}
+                popupAnchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
+                popupTransformOrigin={{ vertical: 'top', horizontal: 'left' }}
+              />
+            ) : null
+          }
+        </Flex>
+        <Flex>
+          <TextInput
+            placeholder={messages.search(selectedPill)}
+            label={messages.search(selectedPill)}
+            value={filterText}
+            onChange={handleFilterChange}
+            size={TextInputSize.SMALL}
+            startIcon={IconSearch}
+          />
+        </Flex>
+      </Flex>
+      {selectedPill === Pills.TRACKS ? (
+        <TracksTab
+          selectedFilter={selectedTrackFilter}
+          filterText={filterText}
+          onClickRow={onClickRow}
+        />
+      ) : (
+        <AlbumsTab
```

Comment:
Minor concern over the name collision with profile `AlbumsTab` but nbd. They're in different paths, but I've noticed too much of this makes global file name search annoying/confusing.

Could go with `DahsboardAlbumsTable` or something